tests/test_metrics/test_neg_p_percent.py

```diff
@@ -0,0 +1,59 @@
+import pandas as pd
+import pytest
+from sklearn.linear_model import LogisticRegression
+from sklearn.pipeline import make_pipeline
+
+from sklego.metrics import neg_p_percent
+from sklego.preprocessing import ColumnSelector
+
+
+@pytest.fixture
+def sensitive_classification_dataset():
+    df = pd.DataFrame({"x1": [1, 0, 1, 0, 1, 0, 1, 1],
+                       "x2": [0, 0, 0, 0, 0, 1, 1, 1],
+                       "y": [1, 1, 1, 0, 1, 0, 0, 0]})
+
+    return df[['x1', 'x2']], df['y']
+
+
+@pytest.fixture
+def sensitive_multiclass_classification_dataset():
+    df = pd.DataFrame({
+        'x1': [1, 0, 1, 0, 1, 0, 1, 1, -2, -2, -2, -2],
+        'x2': [0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 1, 1],
+        'y': [1, 1, 1, 0, 1, 0, 0, 0, 2, 2, 0, 0],
+    })
+    return df[['x1', 'x2']], df['y']
+
+
+def test_neg_p_percent_pandas(sensitive_classification_dataset):
+    X, y = sensitive_classification_dataset
+    mod_unfair = LogisticRegression().fit(X, y)
+    assert neg_p_percent("x2")(mod_unfair, X) == 0
+
+    mod_fair = make_pipeline(
+        ColumnSelector('x1'),
+        LogisticRegression(),
+    ).fit(X, y)
+    assert neg_p_percent("x2")(mod_fair, X) == -0.9
```

Comment:
just to check if i understand. the score for the fair model is ... worse?

Reply:
yes because lower is better in grid search right?

Reply:
i thought higher is better? from the [docs](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GridSearchCV.html) 

![image](https://user-images.githubusercontent.com/1019791/59718034-8386d480-9219-11e9-9acc-a4b8dfd9be64.png)

Reply:
Thanks for catching that one! Inverted the whole thing. Thats what you get for building features without wifi to check the sklearn docs 🙂

Reply:
Might be one for the `import this` feature. 

> Off by one errors are hard to catch, 
> Flipping the sign can be even harder (and worse).